lib/system.ml

```diff
@@ -289,11 +289,16 @@ let with_time ~batch ~header f x =
     Feedback.msg_notice (header ++ str msg ++ fmt_time_difference tstart tend ++ str msg2);
     raise e
 
+let coq_executable_extension ?(byte=Sys.(backend_type = Bytecode)) () =
```

Comment:
The implicit argument doesn't seem to be used anywhere so can be removed.

Reply:
I don't understand what you mean. `byte` is used in an `if` clause and the unit-argument is necessary because the function wouldn't be callable otherwise (compilation error).

Reply:
I mean it's not used by callers.

Reply:
Ah, ok. I misunderstood you. Have implemented your suggestion.